README.md

```diff
@@ -8,22 +8,17 @@ WanderLust is your ultimate travel blog, built with the power of MERN stack. Imm
 
 ## Getting Started
 
-### Setting up the Backend
+### Setting up the Project
 1. Fork this project.
 2. Clone your forked repository.
-3. Navigate to the `backend` directory, run `cd backend`
-4. Run `npm install` to install the required dependencies.
-5. Set up your MongoDB database:
+3. Run `npm run installer`
+4. Set up your MongoDB database:
```

Comment:
I guess here there is a typo because it is `ins` in the `package.json`. And I dont see `installer` in the `package.json`